spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -66,6 +66,7 @@ def create_manager()
           expected_error = /Failure waiting for PowerShell process (\d+) to start pipe server/
           expect(e.message).to match expected_error
           pid = expected_error.match(e.message)[1].to_i
+          sleep(1)
```

Comment:
It would be nice to have a comment here on how we're avoiding a race by calling sleep to wait for the terminated process to complete it's cleanup. As we've discovered, even though `KILL` was called, the PID may not yet be removed from the process table just yet.